src/chains/models.py

```diff
@@ -17,6 +18,11 @@
 sem_ver_validator = RegexValidator(SEM_VER_REGEX, "Invalid version (semver)", "invalid")
 
 
+def native_currency_path(instance: "Chain", filename):
+    _, file_extension = os.path.splitext(filename)  # file_extension includes the dot
+    return f"safe-config/chains/{instance.id}_currency_logo{file_extension}"
```

Comment:
Maybe change this path to `chains/{instance.id}}/currency_logo{extension}`. I might make mandatory that extension is `.png` to prevent issues with the clients